Changelog.md

```diff
@@ -1,3 +1,12 @@
+### 3.5.0 Development
```

Comment:
This is just development, as this could easily be included in a patch release, so we don't necessarily know which is the next version.